hypothesis-python/src/hypothesis/core.py

```diff
@@ -239,7 +239,23 @@ def invalid(message):
         def wrapped_test(*arguments, **kwargs):
             raise InvalidArgument(message)
 
+        def _get_fuzz_target() -> Callable[
+            [Union[bytes, bytearray, memoryview, BinaryIO]], Optional[bytes]
+        ]:
+            def fuzz_one_input(
+                buffer: Union[bytes, bytearray, memoryview, BinaryIO]
+            ) -> Optional[bytes]:
+                return None
+
+            fuzz_one_input.__doc__ = HypothesisHandle.fuzz_one_input.__doc__
+            return fuzz_one_input
+
         wrapped_test.is_hypothesis_test = True
+        wrapped_test.hypothesis = HypothesisHandle(
+            wrapped_test, 
+            _get_fuzz_target, 
```

Comment:
In this case though, I think the fuzz-target-getter should just be the fake `wrapped_test`. Fuzzing an invalid test function isn't going to work, so if the user tries then they should get a sensible exception.

(@Zac-HD knows the fuzzing stuff better than I do, so I'll defer to him on this.)

Reply:
Yep, agree here too.

Specifically we want to raise the exception while setting up the fuzzer (which `_get_fuzz_target=wrapped_test` would do), to avoid wasting a lot of CPU time by "fuzzing" `wrapped_test`.